src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/ProjectProperties/ProjectPropertiesEditorFactory.cs

```diff
@@ -104,8 +113,15 @@ public int MapLogicalView(ref Guid rguidLogicalView, out string? pbstrPhysicalVi
                 return HResult.OK;
             }
 
+            if (rguidLogicalView == DebugPageLogicalViewGuid)
```

Comment:
This shouldn't be hard-coded, it should come from the hierarchy property; VSHPROPID_ProjectPropertiesDebugPageArg

Reply:
I don't think that will be possible here. Seems like there's no hierarchy to query at this point. Am I missing something?

Reply:
The hierarchy was passed to `CreatedEditorInstance`--is that called before this method?

Also note that on the Codespace client the hierarchy will have limited functionality, and it is possible that `VSHPROPID_ProjectPropertiesDebugPageArg` won't work.

Reply:
`MapLogicalView` is called first. Only if it indicates that the view can be mapped will `CreateEditorInstance` be invoked.

The legacy property pages simply accept all logical GUIDs. The first version of this PR did the same. I'll convert this code to a comment explaining the situation.